app/static/src/app/store/sensitivity/state.ts

```diff
@@ -54,5 +54,6 @@ export interface SensitivityState {
     plotSettings: SensitivityPlotSettings,
     result: OdinSensitivityResult | null;
     parameterSetResults: Dict<OdinSensitivityResult>,
-    running: boolean;
+    running: boolean
+    loading: boolean
```

Comment:
I think is the rough explanation for `loading` - might not be obvious what difference is from running...

```suggestion
    loading: boolean // true when sensitivity plot is waiting for plot data
```

Reply:
okay yh so maybe something like:
```typescript
// true only in stochastic mode when odinWorker produces multiple sensitivity
// traces sequentially. This toggle is required to show how many runs out of
// the total have finished.  Turned off when all runs are complete
running: boolean
// true in all modes when user clicks run button, turned off in the allPlotData
// function in sensitivity plot components since they the longest in reshaping
// the data
loading: boolean
```

Reply:
oops, will add that in now